datalad/cmd.py

```diff
@@ -146,14 +146,13 @@ def _readline_rstripped(stdout):
     return stdout.readline().rstrip()
 
 
-class BatchedCommandProtocol(GeneratorMixIn, StdOutErrCapture):
+class BatchedCommandProtocol(StdOutErrCapture, GeneratorMixIn):
```

Comment:
why swap of the order in all (?) cases is needed?  
nothing in the changelog entry hints on that (talks only about removal of `__init__`) and brief descriptions of the commits (now 11) neither.